platform/default/mbgl/storage/offline_database.cpp

```diff
@@ -49,7 +49,8 @@ void OfflineDatabase::ensureSchema() {
             case 1: break; // cache-only database; ok to delete
             case 2: migrateToVersion3(); // fall through
             case 3: migrateToVersion4(); // fall through
```

Comment:
Would it be feasible to smoketest a collection of test fixture databases that go through different subsets of these migration paths?